modules/fastq_tool.py

```diff
@@ -0,0 +1,101 @@
+def count_gc(seq: tuple, gc_bounds: tuple or int = (0, 100)) -> bool:
+    """
+    Function calculates GC composition and if value is in GC composition interval for filtering, it returns True,
+    else False.
+
+    :param seq: tuple of two strings: sequence and quality
+    :param gc_bounds: GC composition interval (in percent) for filtering (default is (0, 100)
+    :return: True if the obtained value is included in gc_bounds, else False
+    """
+    if type(gc_bounds) != tuple:
+        gc_bounds = tuple([0, gc_bounds])
+    gc_result = (seq[0].count('C') + seq[0].count('G')) / len(seq[0]) * 100
+    if gc_result < gc_bounds[0]:
+        return False
+    elif gc_result > gc_bounds[1]:
+        return False
+    else:
+        return True
```

Comment:
А-а-а-а, нееет, не надо так. 
```suggestion
    return gc_bounds[0] <= gc_result <= gc_bounds[1]
```
Вот, красота же)